### PR TITLE
[Osdocs 8065]/[OSDOCS-8893]:Relax GCP organizational policy constraint/Document additional GCP API service requirement

### PR DESCRIPTION
**Pull request by mletalie**
**This PR is for the Shielded VMs constraint policy setting for GCP OSD release that is scheduled for 12/15/23. Do not merge until this date.**

Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-8065 and 

Link to docs preview:

https://68506--docspreview.netlify.app/openshift-dedicated/latest/osd_planning/gcp-ccs#ccs-gcp-customer-procedure_gcp-ccs
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mletalie** [comment]
**REVIEWERS: This change is not to be merged until 12/15/23. DO NOT MERGE UNTIL THAT DATE.**

**Comment by mletalie** [comment]
Hello @lento-sun, 
 Could I please get a review for this PR? Thx!

File: modules/ccs-gcp-customer-procedure.adoc
Comment (renan-campos) [inline modules/ccs-gcp-customer-procedure.adoc:19]:
```suggestion
* `constraints/compute.requireShieldedVm`  (This policy constraint is supported only if the cluster is installed with "Enable Secure Boot support for Shielded VMs" during the initial cluster creation).
```

**Review by rcampos2029 — APPROVED** [review]
I think replacing the double-negative in the constraint description would be an improvement to its readibility, but this is just a suggestion.

**Comment by svmrh** [comment]
@renan-campos - I like your suggestion, but it will make it slightly inconsistent with the note added in the next bullet point. Hence my vote is to keep the current version - 

> This policy constraint is unsupported only if the cluster is installed without selecting "Enable Secure Boot support for Shielded VMs" during the initial cluster creation

Reference: https://68506--docspreview.netlify.app/openshift-dedicated/latest/osd_planning/gcp-ccs#ccs-gcp-customer-procedure_gcp-ccs

image

File: modules/ccs-gcp-customer-procedure.adoc
Comment (skopacz1) [inline modules/ccs-gcp-customer-procedure.adoc:19]: Very inconsequential nit but there's an extra space here:
```suggestion
* `constraints/compute.requireShieldedVm` (This policy constraint is unsupported only if the cluster is installed without selecting "Enable Secure Boot support for Shielded VMs" during the initial cluster creation).
```

**Review by skopacz1 — COMMENTED** [review]
Left a few comments but otherwise LGTM!

File: modules/ccs-gcp-customer-procedure.adoc
Comment (skopacz1) [reply modules/ccs-gcp-customer-procedure.adoc:19]: And I recognize that this would make the wording inconsistent with the next bullet point, but I don't get why that one doesn't say "This policy constraint is supported only after installation." in the first place. Though that's out of scope for my review

Comment (mletalie) [reply modules/ccs-gcp-customer-procedure.adoc:19]: @svmrh,
      Please see comment above. Any thoughts on this suggestion? Sebastian is referring to the constraint as identified in the image below. Thanks! 
image

**Comment by svmrh** [comment]
Thanks @skopacz1 for your review comments. Looks like, many of us feel that we must avoid using double-negative sentences. I am okay to modify the requireShieldedVM note to:

> `constraints/compute.requireShieldedVm`  (This policy constraint is supported only if the cluster is installed with "Enable Secure Boot support for Shielded VMs" selected during the initial cluster creation).

File: rosa-linux.tar.gz
Comment (jeana-redhat) [reply rosa-linux.tar.gz:1]: Followup: this PR is marked as a draft. I think it needs to be converted before merge (I have never encountered a draft PR before so I might not know what I'm talking about :sweat_smile:)

/remove-label merge-review-in-progress
/remove-label merge-review-needed

image

Comment (svmrh) [reply rosa-linux.tar.gz:1]: This feature and PR is about OSD on Google Cloud, therefore not relevant to this PR.

Comment (jeana-redhat) [reply rosa-linux.tar.gz:1]: Thanks for the link! Yeah, I apparently missed whenever this process was added, but it totally makes sense. It sure stopped me in my tracks with the button greyed out :grin:

Comment (mletalie) [reply rosa-linux.tar.gz:1]: > Is this file supposed to be in this PR?

Hi @jeana-redhat,
   Good catch on this extra file. That was very strange! Looks like while downloading file to update the ROSA CLI, the file itself was moved into the openshift-docs repository! Anyway, it is removed so should be good.

Comment (mletalie) [reply rosa-linux.tar.gz:1]: > Thanks for the link! Yeah, I apparently missed whenever this process was added, but it totally makes sense. It sure stopped me in my tracks with the button greyed out 😁

Totally get it! I just happened to catch someone talking about it in a channel the other day and I thought "I will need to remember this at some point!" :)

Comment (jeana-redhat) [reply rosa-linux.tar.gz:1]: LGTM, added the `ok-to-merge` label for later :)

**Pull request merged**
